recipe/meta.yaml

```diff
@@ -303,6 +301,7 @@ outputs:
       requires:
         - {{ compiler('c') }}
         - {{ compiler('cxx') }}
+        - boto3  # [osx and x86]
```

Comment:
the condition of `osx and x86` seems dubious. its either always required (should be in the run section), or just required for testing (should be enabled for all architectures).

Reply:
Hmm, it's not in the root requirements.txt but in some CI requirements. Let me remove the condition for now.